packages/wrangler/src/pages/index.tsx

```diff
@@ -81,6 +82,12 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
 							Publish.Options,
 							Publish.Handler
 						)
+						.command(
+							"tail [deployment-id]",
+							"Tail requests and logs on a deployment",
```

Comment:
phrasing is a bit confusing here IMHO. Maybe smth like `Start a tailing session for a project's deployment and livestream logs from your Functions`. Doesn't have to be this wordy, but u get the gist